pyproject.toml

```diff
@@ -46,6 +46,11 @@ keywords = [
 license = {file = "LICENSE"}
 requires-python = ">=3.10"
 
+[project.optional-dependencies]
+slim = ["numba", "pandas"]
```

Comment:
most of this library doesn't depend on the cli, so there _can_ be a slim option. In fact, a better approach would be to move the dependency on `cltoolbox` to an optional dependency tag called `cli = ["cltoolbox"]`.

Reply:
My vote is to keep cltoolbox part of the default dependencies.  The CLI is the only way I have run HSP2.  A CLI is much easier to use on a cluster or within a parameter estimation run were typically you build out shell scripts or batch files.

Reply:
Ok, I don’t mean to propose we remove a cli option. When I use hsp2 on a cluster I use it as a library as part of another program I’ve written that has its own cli. I guess the question here is if you should have to type `pip install hsp2[cli]` or if I should have to type `pip install hsp2[slim]`. I don’t truly care either way, as long as there is a slim line available.

Of course we could erase the need for this whole conversation by rewriting the 2 command cli in `argparse` which I suggest (and volunteer!) to do.